docs/Reference/Decorators.md

```diff
@@ -114,10 +114,46 @@ fastify.get('/', async function (request, reply) {
 The `dependencies` parameter is an optional list of decorators that the
 decorator being defined relies upon. This list is simply a list of string names
 of other decorators. In the following example, the "utility" decorator depends
-upon "greet" and "log" decorators:
+upon "greet" and "hi" decorators:
 
 ```js
-fastify.decorate('utility', fn, ['greet', 'log'])
+'use strict'
+
+const fastify = require('../fastify')({
+  logger: false
+})
+const fp = require('fastify-plugin')
+
```

Comment:
```suggestion
```

Reply:
I'd remove this. We usually go straightforward on the examples, assuming the user required everything at first.